sympy/physics/control/lti.py

```diff
@@ -0,0 +1,655 @@
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s**2 + p*s + 3, s - 4, s)
+        >>> G1.num
+        p*s + s**2 + 3
+        >>> G2 = TransferFunction((p + 5)*(p - 3), (p - 3)*(p + 1), p)
+        >>> G2.num
+        (p - 3)*(p + 5)
+
+        """
+        return self._num
+
+    @property
+    def den(self):
+        """
+        Returns the denominator polynomial/expression of the transfer function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + 4, p**3 - 2*p + 4, s)
+        >>> G1.den
+        p**3 - 2*p + 4
+        >>> G2 = TransferFunction(3, 4, s)
+        >>> G2.den
+        4
+
+        """
+        return self._den
+
+    @property
+    def var(self):
+        """
+        Returns the variable used to anchor the expressions/polynomials of
+        the transfer function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G1.var
+        p
+        >>> G2 = TransferFunction(0, s - 5, s)
+        >>> G2.var
+        s
+
+        """
+        return self._var
+
+    def _eval_subs(self, old, new):
+        arg_num = self.num.subs(old, new)
+        arg_den = self.den.subs(old, new)
+        argnew = TransferFunction(arg_num, arg_den, self.var)
+        return self if old == self.var else argnew
+
+    def _eval_evalf(self, prec):
+        return TransferFunction(
+            self.num._eval_evalf(prec),
+            self.den._eval_evalf(prec),
+            self.var)
+
+    def _eval_simplify(self, **kwargs):
+        tf = cancel(Mul(self.num, 1/self.den, evaluate=False), expand=False).as_numer_denom()
+        num_, den_ = tf[0], tf[1]
+        return TransferFunction(num_, den_, self.var)
+
+    def expand(self):
+        """
+        Returns the transfer function with numerator and denominator
+        in expanded form.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction((a - s)**2, (s**2 + a)**2, s)
+        >>> G1.expand()
+        TransferFunction(a**2 - 2*a*s + s**2, a**2 + 2*a*s**2 + s**4, s)
+        >>> G2 = TransferFunction((p + 3*b)*(p - b), (p - b)*(p + 2*b), p)
+        >>> G2.expand()
+        TransferFunction(-3*b**2 + 2*b*p + p**2, -2*b**2 + b*p + p**2, p)
+
+        """
+        return TransferFunction(expand(self.num), expand(self.den), self.var)
+
+    def __add__(self, other):
+        if isinstance(other, (TransferFunction, Series)):
+            if not self.var == other.var:
+                raise ValueError("All the transfer functions should be anchored "
+                    "with the same variable.")
```

Comment:
Yes, we can do that. But if the user adds a transfer function to, let's say, an integer.. then it will raise an `AttributeError` stating that it has no attribute `var`. To avoid that, It will raise a `ValueError` at the bottom of the function stating that `TransferFunction` cannot be added with Integer.